setup.py

```diff
@@ -13,4 +13,6 @@
     long_description_content_type="text/markdown",
     url="https://github.com/kkiesling/IsogeomGenerator",
     packages=setuptools.find_packages(),
+    entry_points = {
```

Comment:
PEP 8 recommends no spaces here
```suggestion
    entry_points={
```